### ISSUE TITLE
Consider adding most common image formats to the Template.BinaryExtension list.

### ISSUE THREAD

**Issue by trondr** [PowershellFrameworkCollective/PSModuleDevelopment#175]
Maintaining a complete list of binary extensions is  probably a lost battle, but maybe consider adding the most common image formats?

I set the following before I compile new versions of our templates.

Set-PSFConfig -Module 'PSModuleDevelopment' -Name 'Template.BinaryExtensions' -Value @('.dll', '.exe', '.pdf', '.doc', '.docx', '.xls', '.xlsx','.png','.ico','.bmp','.jpg','.pdb') -Initialize -Description "When creating a template, files with these extensions will be included as raw bytes and not interpreted for parameter insertion."

Just a minor suggestion.

**Comment by FriedrichWeinmann** [comment]
Hi trondr,
sorry for the very, very late response - must have missed that one :(
Very excellent point, there are quite a few common formats I could and should add. Will update it in the next release.

That said, you may want to update your call to this in the meantime:

```powershell
Set-PSFConfig -Module 'PSModuleDevelopment' -Name 'Template.BinaryExtensions' -Value @('.dll', '.exe', '.pdf', '.doc', '.docx', '.xls', '.xlsx','.png','.ico','.bmp','.jpg','.pdb')
```

The `-Initialize` call means it is only applied the first time you initialize - so if you had the module loaded first, it would not apply your settings. Without the `-Initialize` it works the same way both ways - you can update existing configs or pre-seed settings before the module was loaded.

**Comment by FriedrichWeinmann** [comment]
Sorry about the :cough: modest delay.
Implemented in 2.2.11.146
